src/components/instance-select.astro

```diff
@@ -64,7 +64,7 @@ const { instance, errors } = Astro.props;
 		id="remember"
 		name="remember"
 	/>
-	 Remember instance on this device<noscript>.</noscript>
```

Comment:
This breaks the site:

<img width="334" alt="image" src="https://github.com/kytta/share2fedi/assets/6566248/9a2cfe19-5240-4630-950c-8cdaca8c515b">